universal-application-tool-0.0.1/app/services/applicant/JsonPathPredicate.java

```diff
@@ -0,0 +1,22 @@
+package services.applicant;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/** Holds a JsonPath (https://github.com/json-path/JsonPath) path containing a query predicate. */
+public class JsonPathPredicate {
```

Comment:
question (blocking): any reason not to use `@AutoValue` here? Seems like a data class that would benefit from the auto-generated AutoValue features

Reply:
Done